### ISSUE TITLE
bug: Search (platform): tooltip missing from search icon

### ISSUE THREAD

**Issue by Seamoo13** [SAP/fundamental-ngx#4118]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
search icon should have tooltip on hover

![2020-12-16](https://user-images.githubusercontent.com/47522152/102325621-06fac080-3f8c-11eb-8c75-f1674aa56c12.png)

split from: https://github.com/SAP/fundamental-ngx/issues/4038
